cuqi/geometry/_geometry.py

```diff
@@ -16,24 +17,48 @@ def _get_identity_geometries():
 
 class Geometry(ABC):
     """A class that represents the geometry of the range, domain, observation, or other sets.
+
+    It specifies a mapping from the parameter space to the function space (:meth:`par2fun`) and the inverse map if possible (:meth:`fun2par`). The parameters can be for example, the center and width of a hat function, and the function is the resulting hat function evaluated at grid points of a given grid. The geometry keeps track of the dimension and shape of the parameter space (:meth:`par_dim` and :meth:`par_shape`) and the dimension and shape of the function space (:meth:`fun_dim` and :meth:`fun_shape`).
     """
     @property
     @abstractmethod
-    def shape(self):
+    def par_shape(self):
+        """The shape of the parameter space."""
```

Comment:
I agree with you that `par_shape` does not seem to be needed now. It is added to provide the full for options. We can keep it just in case and might remove it in the future.